pkg/secrethub/credentials/encoding.go

```diff
@@ -28,12 +28,12 @@ var (
 )
 
 var (
-	// DefaultCredentialParser defines the default parser for credentials.
-	DefaultCredentialParser = newCredentialParser(DefaultCredentialDecoders)
-	// DefaultCredentialDecoders defines the default list of supported decoders.
-	DefaultCredentialDecoders = []credentialDecoder{rsaPrivateKeyDecoder{}}
-	// DefaultCredentialEncoding defines the default encoding used for encoding credential segments.
-	DefaultCredentialEncoding = base64.URLEncoding.WithPadding(base64.NoPadding)
+	// DefaultParser defines the default parser for credentials.
+	DefaultParser = newParser(DefaultDecoders)
+	// DefaultDecoders defines the default list of supported decoders.
+	DefaultDecoders = []decoder{rsaPrivateKeyDecoder{}}
+	// DefaultEncoding defines the default encoding used for encoding credential segments.
+	DefaultEncoding = base64.URLEncoding.WithPadding(base64.NoPadding)
```

Comment:
I think some of these variables could very well be private?

Reply:
Once again, very true.